scholarx/archive/index.html

```diff
@@ -67,11 +67,18 @@
         <h6>FILTER</h6>
         <hr>
         <!--mentor mentee selection-->
-        <div class="mt-2 mb-5">
-            <select id="selection" class="btn btn-secondary dropdown-toggle" >
-                <option value="mentors">Mentors</option>
-                <option value="mentees">Mentees</option>
-            </select>
+        <div class="mt-2 mb-5" id="selection">
+        <div class="mt-2 mb-5" id="selection">
+            <button class="btn btn-sm custom-checkbox mb-3 py-2 btn-checkbox">
+                <input class="custom-control-input" id="mentorsCheckbox" type="checkbox">
+                <label class="custom-control-label" for="mentors">mentors</label>
```

Comment:
```suggestion
                <label class="custom-control-label" for="mentorsCheckbox">mentors</label>
```
You have to use the id you gave to **"input"** tag for **"for"** attribute.

Reply:
Ok